src/export.c

```diff
@@ -0,0 +1,36 @@
+#define R_NO_REMAP
+#include <Rinternals.h>
+
+#include "export.h"
+
+// Compat for R 3.4.0
+SEXP rlang_MakeExternalPtrFn(DL_FUNC p, SEXP tag, SEXP prot) {
```

Comment:
I realize this is functionally equivalent to the R 3.4.0 definition, but it does have an extra function call, why are we not using the [exact definition from R](https://github.com/wch/r-source/blob/37388191f5493825923b98246ec02e9b13ab1770/src/main/memory.c#L3347-L3356)?